Shared/Samples/Find route in transport network/FindRouteInTransportNetworkView.Model.swift

```diff
@@ -0,0 +1,275 @@
+            routeParameters?.clearStops()
+            
+            // Reset the routes.
+            routeGraphicsOverlay.removeAllGraphics()
+            routeInfo.reset()
+        }
+        
+        /// Adds a stop graphic to the map.
+        /// - Parameter point: The point on the map to add the stop graphic at.
+        private func addStopGraphic(at point: Point) {
+            // Create a text symbol with next index.
+            let textSymbol = TextSymbol(
+                text: "\(stopGraphicsCount + 1)",
+                color: .white,
+                size: 20,
+                horizontalAlignment: .center,
+                verticalAlignment: .middle
+            )
+            textSymbol.offsetY = markerSymbol.offsetY
+            
+            // Create a graphic with the marker symbol and text symbol.
+            let compositeSymbol = CompositeSymbol(symbols: [markerSymbol, textSymbol])
+            let stopGraphic = Graphic(geometry: point, symbol: compositeSymbol)
+            
+            // Add the new graphic to the graphics overlay.
+            stopGraphicsOverlay.addGraphic(stopGraphic)
+        }
+        
+        /// Updates the route parameters stops using graphics in the stop graphics overlay.
+        /// - Parameter indices: A range of indices corresponding to the graphics to create the stops from.
+        private func updateRouteParametersStops(using indices: Range<Int>) {
+            guard indices.lowerBound >= 0 && indices.upperBound <= stopGraphicsCount else { return }
+            
+            // Create a stop for each index in the passed range.
+            var stops: [Stop] = []
+            for i in indices {
+                if let graphicPoint = stopGraphicsOverlay.graphics[i].geometry as? Point {
+                    let stop = Stop(point: graphicPoint)
+                    stops.append(stop)
+                }
+            }
+            
+            // Clear the previous stops from the route parameters.
+            routeParameters?.clearStops()
+            
+            // Set the new stops.
+            routeParameters?.setStops(stops)
+        }
+    }
+    
+    struct RouteInfo {
+        /// The total time in seconds of the combined routes.
+        var totalTime: TimeInterval = 0
+        
+        /// The total distance in meters of the combined routes.
+        var totalDistance = Measurement(value: 0, unit: UnitLength.meters)
+        
+        /// The human-readable label of the route info containing the formatted time and distance.
+        var label: String? {
+            guard totalTime != 0 && totalDistance.value != 0  else { return nil }
+            
+            // Format the time.
+            let totalMinutes = Int(totalTime) / 60
+            let hours = Int(totalMinutes) / 60
+            let minutes = Int(totalMinutes) % 60
+            let hoursText = hours == 0 ? "" : "\(hours) hr "
+            let minutesText = minutes == 0 ? "0 min" : "\(minutes) min"
```

Comment:
Could this just be `let minutesText = "\(minutes) min"`

Reply:
I think it makes more sense to keep it as is since the same pattern was used in the [old iOS implementation](https://github.com/Esri/arcgis-maps-sdk-swift-samples/pull/289) and can be found in Apple and Google Maps. It would have been better to use [formatted()](https://developer.apple.com/documentation/swift/duration/formatted()) here as it would have done this automatically, but `Duration`s are only supported in iOS 16+.

Reply:
This can (and should) be done using a formatter. There may be a simpler way to do it, but one way I found is using a date range and `Date.ComponentsFormatStyle`. I used `DateInterval` as a helper:

![image](https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/2257493/c0a5d23f-5c88-4a3f-bb3f-e294caeefa94)

You could do the same using `totalTime` as the `duration`.